### ISSUE TITLE
Mouse click cell, column or row doesnt appear to have focus outline styling.

### ISSUE THREAD

**Issue by mP1** [mP1/walkingkooka-spreadsheet-react#1205]
- Continuing navigation of cells, columns or rows does have focus outline styling.
- Entering hash token focus outline styling always works.
- Only mouse click does not give focus outline styling, appears element.focus slightly fails.
